deploy/jenkins-ci/Makefile.site.jenkins

```diff
@@ -0,0 +1,60 @@
+# This Makefile is used to automate Kaili website releases.
+# Please, see documentation in RELEASING.adoc
+RELEASE_TYPE ?= 'minor'
+
+SITE_VERSION ?= $(shell sed -rn 's/^VERSION \?= v(.*)/\1/p' Makefile)
+
+# Take the version as-is without filtering out the 'v' (if it exists)
+# This will take 'latest' as is or vX.Y.Z-type.
+VERSION_TAG ?= $(shell sed -rn 's/^VERSION \?= (.*)/\1/p' Makefile)
+
+SITE_GITHUB_URI ?= git@github.com:kiali/kiali.io.git
+SITE_PULL_URI ?= https://api.github.com/repos/kiali/kiali.io/pulls
+SITE_RELEASE_URI = $(subst pulls,releases,$(SITE_PULL_URI))
+SITE_MAIN_BRANCH ?= master
+
+ifeq ($(RELEASE_TYPE),minor)
+  PUSH_CHANGES = y
+
+  # NOTE: The value fo SITE_BUMPED_VERSION and SITE_VERSION_BRANCH will only be valid
+  # after the main Makefile has been modified with the version being released.
+  SITE_BUMPED_VERSION ?= $(shell semver bump minor $(SITE_VERSION))
+  SITE_VERSION_BRANCH ?= $(shell semver bump release $(SITE_VERSION) | sed 's/\.[[:digit:]]\+$$//')
+else
+  PUSH_CHANGES = n
+endif
+
+.PHONY: website-build-archive website-push-version-tag
+.PHONY: all release
+
+all:
+	$(error You must explicitly specify a target)
+
+website-build-archive:
+ifneq ($(PUSH_CHANGES),y)
+	@echo "Will build version: $(SITE_VERSION)"
+	sed -i -r 's/^VERSION \?= v.*/VERSION \?= v$(SITE_BUMPED_VERSION)/' Makefile
+	./scripts/build-archive.sh v$(SITE_VERSION)
+	git add Makefile
+	git add content/documentation/v*
+	git commit -m "Release v$(SITE_VERSION)"
+	git push $(SITE_GITHUB_URI) $$(git rev-parse HEAD):$(SITE_MAIN_BRANCH)
+else
+  @echo "Skipped website release on patch or snapshot releases"
+endif
+
+website-push-version-tag:
+ifeq ($(PUSH_CHANGES),y)
+	git push $(SITE_GITHUB_URI) $$(git rev-parse HEAD):refs/tags/v$(SITE_VERSION)
+ifdef GH_TOKEN
+	@echo "Creating GitHub release..."
+	@curl -H "Authorization: token $(GH_TOKEN)" \
+	  -H "Content-Type: application/json" \
+	  -d '{"name": "Kiali.io $(SITE_VERSION)", "tag_name": "$(VERSION_TAG)"}' \
+	  -X POST $(SITE_RELEASE_URI)
+endif
```

Comment:
I think we don't need to create releases for kiali.io repository
It's OK to create them, but I'd prefer to not :)